src/main/java/org/apache/commons/lang3/annotation/ToEqualsExclude.java

```diff
@@ -0,0 +1,15 @@
+package org.apache.commons.lang3.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * Use this annotation to builds a equals excluding the annotated field.
+ */
+@Retention(RUNTIME)
+@Target(ElementType.FIELD)
+public @interface ToEqualsExclude {
```

Comment:
@FelipeAdorno I don't understand our comment. Do you see a use case for using any of the new annotations without using the corresponding builder?